.bashrc

```diff
@@ -0,0 +1 @@
+export PATH=$HOME/bin:/usr/local/bin:node_modules/.bin:$PATH
```

Comment:
this should not be here.

Reply:
@rowasc, Here's the ticket link https://github.com/ushahidi/platform/issues/3906

Reply:
> this should not be here.

let me clean it up

Reply:
ah thanks @RachealN
@noone0212 had sent a PR for a similar issue (or the same) but I see he closed it. @noone0212 can you let us know if you'd want to bring that back?

Reply:
Yes, @rowasc I would like to bring that back, the issue occurred during rebuilding the platform. I'm so sorry, my bad.

Reply:
Great
@rowasc can you check this issue https://github.com/ushahidi/platform/issues/3918
i suggested some approach, kindly check it and if you approve, i start to work on it